docs/team/khinkhinn.md

```diff
@@ -0,0 +1,54 @@
+  * Credits: *{mention here if you reused any code/ideas from elsewhere or if a third-party library is heavily used in the feature so that a reader can make a more accurate judgement of how much effort went into the feature}*
+  
+* **Feature**: Added the ability to add tags.
+  * What it does: allows the user to add a new tag with zero contacts in the tag. 
+  * Justification: This feature allows for more flexibility of how the app is being used as users can create additional tags that may not necessarily be related to any contacts. A possible example might be the user having a `personal` tag, so that the user can add personal tasks and or meeting links to the task. 
+  
+* **Feature**: Added the ability to add tasks to tags.
+  * What it does: allows the user to add tasks to tags one at a time.
+  * Justification: This feature improves the product significantly because the functionality of tags has been greatly improved. Tags are now capable of storing more information, and tasks in particular can help the user be more organised.
+  * Highlights: This enhancement was hard to implement as adding a task was more like editing a tag. Previous edit commands did not care about the previous entry, and merely overwrote them (such as tags being rewritten when user edits a contact). However for task related commands, it was necessary to edit the previous list of tasks, and this posed a challenge. We managed to solve this by introducing a new method that would work with the current implementation.
+
+* **Feature**: Added the ability to delete tasks from tags.
+  * What it does: allows the user to delete tasks from tags one at a time. This is done by using both the index of the tag and the index of the task.
+  * Justification: This feature improves the product as the user can now delete tasks that they have either added accidentally or made a typo in.
+  * Highlights: Similar to the above feature, we initially struggled with how to use the currently implemented model to add this new feature. However we managed to come up with a work around to the problem of finding the exact task to delete by implementing an alphabetical index, so that it would be easier to identify which index to delete.
+
+### Enhancements to Existing Features
+* **Enhancement**: Refactored the code to use terms specific to our app (Pull requests [\#53]())
+    * Justification: This improved readability of code for other developers, as our app was adapted from an app called "AddressBook", which might be confusing for new developers unaware of this. Hence refactoring the code to use the term "Projact" instead of "AddressBook" in various files made the code more consistent.
+* **Enhancement**: Fixed bug that resulted in text being cut off when it was longer than the result display box. (Pull requests [\#234]())
+
+### Code Contributed
+* [RepoSense link]()
```

Comment:
```suggestion
* [RepoSense link](https://nus-cs2103-ay2021s1.github.io/tp-dashboard/#breakdown=true&search=khinkhinn&sort=groupTitle&sortWithin=title&since=2020-08-14&timeframe=commit&mergegroup=&groupSelect=groupByRepos&checkedFileTypes=docs~functional-code~test-code~other)
```